tests/test_cli.py

```diff
@@ -26,5 +34,21 @@ def test_cli_headers():
     ]
 
 
-class App:
-    pass
+@pytest.mark.skipif(sys.platform == "win32", reason="require unix-like system")
```

Comment:
```suggestion
@pytest.mark.filterwarnings(r"ignore:unclosed .*:ResourceWarning")
```

Reply:
The suggestion was supposed to be an addition, not a replacement, right?

Reply:
no this should be a replacement